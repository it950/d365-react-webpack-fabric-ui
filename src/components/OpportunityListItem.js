import React from 'react'

const OpportunityListItem = ({ opportunity }) => {
  /*eslint no-undef: 0*/
  // const url = Xrm.Page.context.prependOrgName(
  //   `/main.aspx?pagetype=entityrecord&etn=opportunity&id=${opportunity.opportunityid}`,
  // )
  return (
    <li>
      <p>
        <a href="#" target="_blank">
          {opportunity.name}
        </a>
      </p>
      <p>Created On {opportunity.createdon}</p>
    </li>
  )
}

export default OpportunityListItem
