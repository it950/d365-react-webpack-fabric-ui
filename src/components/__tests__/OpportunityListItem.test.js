import React from 'react'
import { shallow } from 'enzyme'

import OpportunityListItem from '../OpportunityListItem'
import opportunities from '../../fixtures/opportunities.js'

describe('OpportunityListItem', () => {
  it('should render correctly', () => {
    const props = { opportunity: opportunities[0] }
    const wrapper = shallow(<OpportunityListItem {...props} />)
    expect(wrapper).toMatchSnapshot()
  })
})
