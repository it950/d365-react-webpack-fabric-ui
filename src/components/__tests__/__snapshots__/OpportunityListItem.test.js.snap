// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OpportunityListItem should render correctly 1`] = `
<li>
  <p>
    <a
      href="#"
      target="_blank"
    >
      Big Sale
    </a>
  </p>
  <p>
    Created On 
    2017-9-21 00:00:00
  </p>
</li>
`;
